plugins/scaffolder-react/src/next/components/TemplateCard/TemplateCard.tsx

```diff
@@ -15,81 +15,42 @@
  */
 
 import { RELATION_OWNED_BY } from '@backstage/catalog-model';
-import { MarkdownContent, UserIcon } from '@backstage/core-components';
-import {
-  IconComponent,
-  useAnalytics,
-  useApp,
-} from '@backstage/core-plugin-api';
-import {
-  EntityRefLinks,
-  getEntityRelations,
-} from '@backstage/plugin-catalog-react';
+import { IconComponent, useAnalytics } from '@backstage/core-plugin-api';
+import { getEntityRelations } from '@backstage/plugin-catalog-react';
 import { TemplateEntityV1beta3 } from '@backstage/plugin-scaffolder-common';
-import Box from '@material-ui/core/Box';
 import Card from '@material-ui/core/Card';
 import CardActions from '@material-ui/core/CardActions';
 import CardContent from '@material-ui/core/CardContent';
-import Chip from '@material-ui/core/Chip';
 import Divider from '@material-ui/core/Divider';
-import Button from '@material-ui/core/Button';
 import Grid from '@material-ui/core/Grid';
 import { makeStyles, Theme } from '@material-ui/core/styles';
-import LanguageIcon from '@material-ui/icons/Language';
 import React, { useCallback } from 'react';
 import { CardHeader } from './CardHeader';
-import { CardLink } from './CardLink';
 import { usePermission } from '@backstage/plugin-permission-react';
 import { taskCreatePermission } from '@backstage/plugin-scaffolder-common/alpha';
+import { TemplateCardContent } from './TemplateCardContent';
+import { TemplateCardTags } from './TemplateCardTags';
+import { TemplateCardLinks } from './TemplateCardLinks';
+import { TemplateCardActions } from './TemplateCardActions';
 
-const useStyles = makeStyles<Theme>(theme => ({
-  box: {
-    overflow: 'hidden',
-    textOverflow: 'ellipsis',
-    display: '-webkit-box',
-    '-webkit-line-clamp': 10,
-    '-webkit-box-orient': 'vertical',
-  },
-  markdown: {
-    /** to make the styles for React Markdown not leak into the description */
-    '& :first-child': {
-      margin: 0,
-    },
-  },
-  label: {
-    color: theme.palette.text.secondary,
-    textTransform: 'uppercase',
-    fontWeight: 'bold',
-    letterSpacing: 0.5,
-    lineHeight: 1,
-    fontSize: '0.75rem',
-  },
-  footer: {
-    display: 'flex',
-    justifyContent: 'space-between',
-    flex: 1,
-    alignItems: 'center',
-  },
-  ownedBy: {
-    display: 'flex',
-    alignItems: 'center',
-    flex: 1,
-    color: theme.palette.link,
-  },
+const useStyles = makeStyles<Theme>(() => ({
+  actionContainer: { padding: '16px', flex: 1, alignItems: 'flex-end' },
 }));
 
+/** @alpha */
+export interface TemplateCardAdditionalLink {
+  icon: IconComponent;
+  text: string;
+  url: string;
+}
+
 /**
  * The Props for the {@link TemplateCard} component
  * @alpha
  */
 export interface TemplateCardProps {
   template: TemplateEntityV1beta3;
-  additionalLinks?: {
```

Comment:
Can we keep this type inline for now? Don't really wanna add too much more API surface exposed out of the package :pray:

Reply:
Sure thing, you're the boss :)